homeassistant/components/tasmota/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
It will be time to introduce a new Integration MQTT discovery type so that the MQTT integration can notify other integrations if it finds data at topics that those integrations can manage.

So in here we would do `"mqtt_discovery": "tasmota/#"` and if any message comes in, the MQTT integration starts the Tasmota config flow.

Reply:
It's a great idea and I didn't forget about it!
Let's add MQTT integration discovery in a separate PR though?

Reply:
Yesss